h/accounts/models.py

```diff
@@ -130,6 +130,11 @@ def is_activated(self):
     _password = sa.Column('password', sa.UnicodeText(), nullable=False)
     # Password salt
     salt = sa.Column(sa.UnicodeText(), nullable=False)
+    # Last password update
+    last_password_update = sa.Column(sa.DateTime(timezone=False),
+                                     default=sa.sql.func.now(),
+                                     server_default=sa.func.now(),
+                                     nullable=True)
```

Comment:
This shouldn't be nullable here. The idea is that you can add the metadata in the model as non-nullable, and then add a nullable column with the migration.